### PR TITLE
Localization for signals example

### PR DESCRIPTION
**Pull request by kgoo124**
See issue #111

### CODE REVIEW

File: basics/signals/src/index.ts [hunk 1]
```diff
@@ -43,11 +45,13 @@ function activate(
   app: JupyterFrontEnd,
   palette: ICommandPalette,
   mainMenu: IMainMenu,
-  launcher: ILauncher | null
+  launcher: ILauncher | null,
+  translator: ITranslator
```
Comment (goanpeca) [inline basics/signals/src/index.ts:49]: Hi @kgoo124 maybe the I translator should go before the "optional" parameter.

`ILauncher | null` I guess this means ILauncher can be `undefined?

Maybe this:
```ts
  translator: ITranslator,
 launcher: ILauncher | null
```

Reads better?

**Review by goanpeca — COMMENTED** [review]
Left a minor comment/question, mostly stylistic.

Otherwise looks good.

Thanks a lot for working on this @kgoo124 !

**Comment by jtpio** [comment]
@kgoo124 should we target 3.0 for this PR too? And check the `embedme` snippets still match, similar to https://github.com/jupyterlab/extension-examples/pull/112?

Thanks!

**Comment by kgoo124** [comment]
Yes, will do! Working on the other PR's and will let you know when they are ready for merge

File: basics/signals/src/index.ts (hunk 1 above)
Comment (kgoo124) [reply basics/signals/src/index.ts]: Fixed!

**Comment by kgoo124** [comment]
@jtpio ready for merge

**Comment by kgoo124** [comment]
Looks like I'm getting this linting error. The prettier pre-commit adds this in, but eslint doesn't like it. What should I do here?

**Comment by jtpio** [comment]
@kgoo124 could it be that the file was formatted from the code editor, which might be using a different version of prettier?

I ran `jlpm run prettier` from the top level locally and pushed https://github.com/jupyterlab/extension-examples/pull/116/commits/b48be4a5df67891f56d5604161e46eb3a3937f0b, which should fix the linting issue.

**Comment by jtpio** [comment]
Thansk @kgoo124, looks good.

The server extension failures seem unrelated to this change, opened https://github.com/jupyterlab/extension-examples/issues/124 to track it.

**Review by jtpio — APPROVED** [review]

**Pull request merged**